test/integration/features/basic.feature

```diff
@@ -60,7 +60,7 @@ Feature: Basic test
     
     @darwin @linux @windows
     Scenario: CRC status check
-        When with up to "10" retries with wait period of "1m" command "crc status" output should not contain "Stopped"
+        When with up to "15" retries with wait period of "1m" command "crc status" output should not contain "Stopped"
```

Comment:
This change seems unrelated?

Reply:
Yes, completely unrelated - I just noticed that the PR was not getting through because this step fails. We discussed it on Slack today - some operators take time to come up. They are needed in the next steps, so longer wait seems reasonable. It didn't seem big enough change to warrant a separate PR - was I wrong? Or should I mention it in the commit?

Reply:
I'd just split it, but mentioning it in the commit log would be good enough so that people know this was not an oversight

Reply:
OK, it's in the log now - next time I'll opt for a separate PR! Thanks for reviewing :)